src/account_manager.rs

```diff
@@ -608,8 +608,8 @@ mod tests {
                     .with_parent2(MessageId::new([0; 32]))
                     .with_payload(Payload::Indexation(Box::new(Indexation::new(
                         "".to_string(),
```

Comment:
I think the test is failing because empty indexes are no longer allowed (https://github.com/Alex6323/bee-p/blob/1ad65ed0909bf2cc55c3b61e16d7bf0b1f124f7f/bee-message/src/payload/indexation.rs#L35)

Reply:
yeah I realised that on the tests branch but forgot to add it here, thanks!

Reply:
I changed `"".to_string()` to `"a".to_string()` and now it's failing in another place: 

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: MissingField("network_id")', src/account_manager.rs:614:22
```